client.py

```diff
@@ -61,7 +61,7 @@ def handle_connection(self, server):
                 raise Exception(f"This client (ID %s) was already assigned a player ID!", self.player_id)
 
             elif parsed_message.code == MessageType.JEOPARDY_QUESTION:
-                [player_id, jeopardy_category, tile] = parsed_message.args  # TODO: Why is this client receiving its own player ID?
+                [jeopardy_category, tile] = parsed_message.args
```

Comment:
I would leave `playerID` here so the client can figure out if it is currently their turn and then only display the "submit" button if it is their turn.